src/app/core/components/header/header.component.ts

```diff
@@ -122,4 +123,13 @@ export class HeaderComponent implements OnInit {
         }
       });
   }
+
+  openMenuDialog(): void {
+    if (this.authState.isLoggedIn) {
```

Comment:
i think we won't need this check. 
because the menu should open either way, if you are logged in or not